SwiftWisdomTests/NSDataTrimTests.swift

```diff
@@ -13,28 +13,20 @@ import XCTest
 class NSDataTrimTests: XCTestCase {
     let ProductName = "Hello, Headphone Name!".data(using: String.Encoding.utf8)!
     let ProductInfoProductNameRawPacketPrefix = "<0102a616>"
-        .ip_dataFromHexadecimalString()!
+        .ip_dataFromHexadecimalString() as! Data
     
     let extraAppendedData = "2342 afa2 ffed 2222"
-        .ip_dataFromHexadecimalString()!
-    
-    lazy var ProductInfoProductNameRawPacketData: Data = {
-        let mutable = NSMutableData()
-        mutable.append(self.ProductInfoProductNameRawPacketPrefix)
-        mutable.append(self.ProductName)
-        mutable.append(self.extraAppendedData)
-        return (NSData(data: mutable as Data) as Data)
-    }()
+        .ip_dataFromHexadecimalString() as! Data
     
     func testTrimRange() {
-        let data = "aa 11 11 11 11 11 bb".ip_dataFromHexadecimalString()!
-        let mutable = NSData(data: data) as Data
+        let data = "aa 11 11 11 11 11 bb".ip_dataFromHexadecimalString()
+        let mutable = NSMutableData(data: data as! Data)
```

Comment:
Yeah, because the data is optional here. But you make a good point, so I'm forcing unwrapping in the line above. And this can be downcast here.